src/replication/log_generator.cpp

```diff
@@ -133,8 +133,16 @@ namespace cubreplication
   void
   log_generator::append_repl_object (replication_object &object)
   {
+#define MAX_PACKABLE_ENTRIES 1000
```

Comment:
Use `const size_t` variable declaration; please consider making it a static variable of log_generator class.

Other than that it looks good to me.